dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_impl.cuh

```diff
@@ -98,18 +98,20 @@ template <bool horz_subsample, bool vert_subsample, typename T>
 __inline__ __device__
 void ycbcr_to_rgb_subsampled(ivec2 offset, const Surface2D<uint8_t>& out,
                              YCbCrSubsampled<T, horz_subsample, vert_subsample> ycbcr) {
-  using kernels::color::jpeg::ycbcr_to_rgb;
+  auto ycbcr_to_rgb = [](auto x) {
```

Comment:
why this change?

Reply:
Because it's no longer a namespace, so "using" doesn't work.